### ISSUE TITLE
Error when generating tracking path

### ISSUE THREAD

**Issue by eteubert** [podlove/radiator#208]
Looks like some missing preloads for the tracking url generation.

```
radiator_1  | 13:33:46.632 request_id=FcCiZWNhB4mA5XsAAAIh [info] POST /api/rest/v1/audios/13/audio_files
email_1     | [APIv1] KEEPALIVE /api/v1/events
radiator_1  | 13:33:47.984 request_id=FcCiZWNhB4mA5XsAAAIh [info] Sent 500 in 1352ms
radiator_1  | 13:33:47.984 request_id=FcCiZWNhB4mA5XsAAAIh [info] Converted error ArgumentError to 500 response
radiator_1  | 13:33:47.989 [error] #PID<0.5908.0> running RadiatorWeb.Endpoint (connection #PID<0.5893.0>, stream id 4) terminated
radiator_1  | Server: localhost:4000 (http)
radiator_1  | Request: POST /api/rest/v1/audios/13/audio_files
radiator_1  | ** (exit) an exception was raised:
radiator_1  |     ** (ArgumentError) cannot convert nil to param
radiator_1  |         (phoenix) lib/phoenix/param.ex:67: Phoenix.Param.Atom.to_param/1
radiator_1  |         (radiator) RadiatorWeb.Router.Helpers.tracking_path/6
radiator_1  |         (radiator) lib/radiator_web/router.ex:1: RadiatorWeb.Router.Helpers.tracking_url/6
radiator_1  |         (radiator) lib/radiator_web/views/api/audio_file_view.ex:31: RadiatorWeb.Api.AudioFileView.render/2
radiator_1  |         (phoenix) lib/phoenix/view.ex:399: Phoenix.View.render_to_iodata/3
radiator_1  |         (phoenix) lib/phoenix/controller.ex:729: Phoenix.Controller.__put_render__/5
radiator_1  |         (phoenix) lib/phoenix/controller.ex:746: Phoenix.Controller.instrument_render_and_send/4
radiator_1  |         (radiator) lib/radiator_web/controllers/api/audio_file_controller.ex:1: RadiatorWeb.Api.AudioFileController.action/2
```

**Comment by eteubert** [comment]
Also happens in other scenarios

```
radiator_1  | 14:06:56.768 request_id=FcCkNL6kHbA53o0AAAFE [info] POST /api/graphql
radiator_1  | 14:06:56.852 request_id=FcCkNL6kHbA53o0AAAFE [info] Sent 500 in 84ms
radiator_1  | 14:06:56.852 request_id=FcCkNL6kHbA53o0AAAFE [info] Converted error ArgumentError to 500 response
radiator_1  | 14:06:56.856 [error] #PID<0.5352.0> running RadiatorWeb.Endpoint (connection #PID<0.5350.0>, stream id 2) terminated
radiator_1  | Server: localhost:4000 (http)
radiator_1  | Request: POST /api/graphql
radiator_1  | ** (exit) an exception was raised:
radiator_1  |     ** (ArgumentError) cannot convert nil to param
radiator_1  |         (phoenix) lib/phoenix/param.ex:67: Phoenix.Param.Atom.to_param/1
radiator_1  |         (radiator) RadiatorWeb.Router.Helpers.tracking_path/6
radiator_1  |         (radiator) lib/radiator_web/router.ex:1: RadiatorWeb.Router.Helpers.tracking_url/6
radiator_1  |         (radiator) lib/radiator_web/graphql/admin/resolvers/editor.ex:380: RadiatorWeb.GraphQL.Admin.Resolvers.Editor.get_file_url/3
radiator_1  |         (absinthe) lib/absinthe/resolution.ex:209: Absinthe.Resolution.call/2
radiator_1  |         (absinthe) lib/absinthe/phase/document/execution/resolution.ex:209: Absinthe.Phase.Document.Execution.Resolution.reduce_resolution/1
radiator_1  |         (absinthe) lib/absinthe/phase/document/execution/resolution.ex:168: Absinthe.Phase.Document.Execution.Resolution.do_resolve_field/4
radiator_1  |         (absinthe) lib/absinthe/phase/document/execution/resolution.ex:153: Absinthe.Phase.Document.Execution.Resolution.do_resolve_fields/6
```

**Comment by eteubert** [comment]
Fixed in eeaf2183bcf55b842cd0efbc1c0bc99d1bf02ccc
